communities/src/lib.rs

```diff
@@ -55,10 +55,8 @@ pub mod pallet {
 	use frame_system::pallet_prelude::*;
 
 	const STORAGE_VERSION: StorageVersion = StorageVersion::new(1);
-
 	#[pallet::pallet]
 	#[pallet::storage_version(STORAGE_VERSION)]
-	#[pallet::generate_store(pub (super) trait Store)]
```

Comment:
Same here, generate store is needed, although I can't tell you all its semantics :P

Reply:
Beautiful, thanks for the comment. I was sure this is not the case. XD